packages/app/src/components/Vote/Vote.tsx

```diff
@@ -9,8 +9,102 @@ import {
 import LinearGradient from 'react-native-linear-gradient';
 import { colors, Card } from '../UI';
 import { useNavigation } from '@react-navigation/native';
+import { graphql } from 'relay-runtime';
+import { usePagination } from 'relay-hooks';
+import { Vote_query, Vote_query$key } from './__generated__/Vote_query.graphql';
+import { VotePaginationQueryVariables } from './__generated__/VotePaginationQuery.graphql';
+
+const votesFragmentSpec = graphql`
+    fragment Vote_query on Query
+    @argumentDefinitions(
+        count: { type: "Int", defaultValue: 20 }
+        cursor: { type: "String" }
+    ) {
+        challenges(first: $count, after: $cursor)
+            @connection(key: "Vote_challenges", filters: []) {
+            endCursorOffset
+            startCursorOffset
+            count
+            pageInfo {
+                hasNextPage
+                hasPreviousPage
+                startCursor
+                endCursor
+            }
+            edges {
+                node {
+                    _id
+                    title
+                    image
+                    creator {
+                        username
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const connectionConfig = {
+    getVariables(
+        props: Vote_query,
+        { count, cursor }: VotePaginationQueryVariables
+    ) {
+        return {
+            count,
+            cursor
+        };
+    },
+    query: graphql`
+        # Pagination query to be fetched upon calling 'loadMore'.
+        # Notice that we re-use our fragment, and the shape of this query matches our fragment spec.
+        query VotePaginationQuery($count: Int!, $cursor: String) {
+            ...Vote_query @arguments(count: $count, cursor: $cursor)
+        }
+    `
+};
+
+type Props = {
+    query: Vote_query$key;
+};
+
+export const Vote = (props: Props): ReactElement => {
+    const [isFetchingTop, setIsFetchingTop] = useState(false);
+    const [
+        query,
+        { isLoading, hasMore, loadMore, refetchConnection }
+    ] = usePagination(votesFragmentSpec, props.query);
+
+    console.log('xzxz', query);
```

Comment:
remove this

Reply:
Hello @youngkidwarrior, this is an outdated file. This has already been removed.

https://github.com/ShenaniganDApp/shenanigan-monorepo/pull/354/files#diff-884dea3a4bc33a01cf5995099a77771a1cee37cbdebd7e2f54a078bb67454619R78